public/canfarRootCopy/cadc.user.js

```diff
@@ -0,0 +1,308 @@
+; (function ($, window, undefined) {
```

Comment:
Does this file differ from https://www.cadc-ccda.hia-iha.nrc-cnrc.gc.ca/static/js/cadc/cadc.user.js?  It can just be used instead if not.

Reply:
It does vary - there's a baseURL value in the inputs object that is used in the this.loadCurrent to allow the whoami call to be pointed to a useful location, rather than just go to the local host. For the dev environment, there's no dev-www.canfar.net/access/proxy. 

The goal for this pass was to not change the canfar-root code, so even though this is a small difference, testing scope would balloon if it was added there.

Reply:
baseURL is a value that already existed in science portal that was used elsewhere.